assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -51,9 +51,9 @@ angular.module("openshiftConsole")
       DataService.get("imagestreams", scope.imageName, scope, {namespace: scope.namespace}).then(function(imageRepo){
           scope.imageRepo = imageRepo;
           var imageName = scope.imageTag;
-          DataService.get("imagestreamtags", imageRepo.metadata.name + ":" + imageName, {namespace: scope.namespace}).then(function(image){
-              scope.image = image;
-              angular.forEach(image.dockerImageMetadata.ContainerConfig.Env, function(entry){
+          DataService.get("imagestreamtags", imageRepo.metadata.name + ":" + imageName, {namespace: scope.namespace}).then(function(imageStreamTag){
+              scope.image = imageStreamTag.image;
+              angular.forEach(imageStreamTag.image.dockerImageMetadata.ContainerConfig.Env, function(entry){
```

Comment:
From what I can tell, he moved `dockerImage*` under a new `image` property to the image stream tag. Impact here and to the image stream resolver below.

https://github.com/openshift/origin/issues/2851#issuecomment-109317735

@ncdc Does my change look right?